config/webhook/manifests.yaml

```diff
@@ -1,5 +1,3 @@
-# Originally generated via:
```

Comment:
Can we add this comment back?  And should we update the proper command to run to generate this file.  I think some of these parms were removed.